packages/components/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as Heading } from "./Heading";
+export { default as TestComponent } from "./TestComponent";
```

Comment:
This line is generated from the script for the purpose of reviewing. TODO: remove this file before merging this PR.

NOTE: This generated line is not automatically alphabetized, so, after running the script, you need to manually move the import to the right line.

Reply:
oh aw, the `sortIndex` function in `plopfile.js` is supposed to automatically do the sorting 😥

Reply:
Nvm, I checked again and the sorting does work! Maybe I was thinking of the imports in `TestComponent.stories.tsx`? 🤷